library/profiler_builtins/build.rs

```diff
@@ -50,6 +50,13 @@ fn main() {
         cfg.flag("-fno-builtin");
         cfg.flag("-fvisibility=hidden");
         cfg.flag("-fomit-frame-pointer");
+        if cfg.get_compiler().path().to_str() == Some("gcc")
+            || cfg.get_compiler().path().to_str() == Some("cc")
+        {
+            cfg.flag("-Wno-prio-ctor-dtor");
```

Comment:
`cfg.flag_if_supported("-Wno-prio-ctor-dtor");` should be a more robust alternative to checking for gcc / cc.

Reply:
done